### PR TITLE
Linking class exercises with the slides

### PR DESCRIPTION
**Pull request by AnnaDinaburgVulikh**
- Changed the corresponding sections to include a reference to the exercise file.
- Renamed the class exercises to the proper convention (using exe abreviation returns an error)
- Updated the exercises markdown to the slides formating

### CODE REVIEW

File: docs/course_materials/control_structures.html [hunk 1]
```diff
@@ -205,14 +205,7 @@ <h3><b>if-elif-else Statements</b></h3>
...
 
-                <section>
-                    <h3><b>Let's Practice together!</b></h3>
-                    <ul>
-                      <li>Create a file named <b>is_positive.py</b></li>
-                      <li>In this file, write a program that defines parameter 'a'.</li>
-                      <li>The program checks if 'a' is positive, negative or equal zero and prints a proper message. </li>
-                    </ul>
-                    <img data-natural-width="340" data-natural-height="234" style="" data-src="https://posteo.de/images/daten.png" />
+                <section  data-markdown="exercises/05_Control_structures/class_exercise_1.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:208]: <section  data-markdown=...>
Extra space
(Apply for all occurrences)

File: docs/course_materials/exercises/05_Control_structures/class_exercise_1.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### Let's Practice together!
+
+- Create a file named *is_positive.py*
+- In this file, Write a program that defines parameter *'a'*.
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/class_exercise_1.md:4]: Write -> write

**Review by sleviim — CHANGES_REQUESTED** [review]
Thanks, @AnnaDinaburgVulikh!
We need to decide what syntax suits the best for the exercises' headline, 
and change all headlines corresponding to that. 
@shiramax, what do you think?

**Comment by shiramax** [comment]
@AnnaDinaburgVulikh thanks for the PR! good job!
I agree with @sleviim thank the titles should be in the same format in all the files.
how about:  Exercise 1 - Let's Practice together!

also, can you please add a README file that explains how to run the server (since it is new we nee to run it)

**Review by shiramax — CHANGES_REQUESTED** [review]
@AnnaDinaburgVulikh thanks for the PR! good job!
I agree with @sleviim thank the titles should be in the same format in all the files.
how about: Exercise 1 - Let's Practice together!

also, can you please add a README file that explains how to run the server (since it is new we nee to run it)

**Comment by sleviim** [comment]
I think that "Exercise 1 - Let's Practice together!" might be an over-kill
(If it's an exercise, we're practicing it :))

What about 
Let's Practice together #1  ?

**Comment by shiramax** [comment]
@sleviim fine by me :) thanks!

**Comment by AnnaDinaburgVulikh** [comment]
> an you please add a README file that explains how to run the server (since it is new we nee to run it)

@shiramax I was using a `pipenv shell` and `python -m http.server 9000` to create a local server in order to test that the slides really work.
The assumption was that since the slides run from the red-hat site, it acts like a server.
I can add a README for running the slides from a local machine for development use (mainly the 2 snippets above). Is that what you meant?

**Comment by AnnaDinaburgVulikh** [comment]
Made the fixes you requested.

**Comment by AnnaDinaburgVulikh** [comment]
> Please try to use xdg-open in order to test the slides.

The xdg-open command only opens a file/url.
In order to run the slides with this fix you need a server to run on.
It can work by running `python -m http.server 9000` and then `xdg-open http://localhost:9000/docs/course_materials/python_intro.html`

**Comment by sleviim** [comment]
The xdg-open is a great way for reviewing your own slides changes before committing them.
But
When you need to generate another source on your slide (the new md file you've added), the generator-reveal requires using an http server, and not using a local file:// as xdg-open does. 

LGTM

**Comment by rollandf** [comment]
We need to make sure that github pages renders this OK.
Can you please test it?

**Comment by AnnaDinaburgVulikh** [comment]
@rollandf Tested it on my fork, it works. Here is the link: https://annadinaburgvulikh.github.io/ROSE/linux_intro.html

**Comment by rollandf** [comment]
> @rollandf Tested it on my fork, it works. Here is the link: https://annadinaburgvulikh.github.io/ROSE/linux_intro.html

Thanks!

File: docs/course_materials/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Create an `.md` exercise file. Please follow the styling conventions.  
+
+2. Add a new section in the corresponding `.html` using the following:  
+```html
+<section data-markdown="relative/path/class_exercise_number.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
+</section>
+```
+   - Example path: `"exercises/01_Linux/class_exercise_2.md"`
+   - Make sure not to use the abriviation 'exe', the styling of the slides won't work propperly.
```
Comment (rollandf) [inline docs/course_materials/README.md:12]: abriviation -> abbreviation
propperly -> properly

**Review by rollandf — CHANGES_REQUESTED** [review]
Some minor typos.
Otherwise looks good! 
Thanks

File: docs/course_materials/README.md (hunk 3 above)
Comment (AnnaDinaburgVulikh) [reply docs/course_materials/README.md]: Fixed it and added an option to test by using `xdg-open`.

**Review by rollandf — APPROVED** [review]

**Pull request merged**